rest_server.go

```diff
@@ -154,29 +156,32 @@ func (a *restServer) installRoutes(routesInfo map[int][]RouteInfo) {
 	}
 
 	// non versioned routes
-	a.multiplexer.Get("/:category/:id", a.makeHandler(handleRetrieve))
-	a.multiplexer.Put("/:category/:id", a.makeHandler(handleUpdate))
-	a.multiplexer.Patch("/:category/:id", a.makeHandler(handlePatch))
-	a.multiplexer.Delete("/:category/:id", a.makeHandler(handleDelete))
-	a.multiplexer.Get("/:category", a.makeHandler(handleRetrieveMany))
-	a.multiplexer.Get("/:parentcategory/:id/:category", a.makeHandler(handleRetrieveMany))
-	a.multiplexer.Post("/:category", a.makeHandler(handleCreate))
-	a.multiplexer.Post("/:parentcategory/:id/:category", a.makeHandler(handleCreate))
-	a.multiplexer.Head("/:category", a.makeHandler(handleInfo))
-	a.multiplexer.Head("/:parentcategory/:id/:category", a.makeHandler(handleInfo))
+	a.multiplexer.Get(a.cfg.restServer.apiPrefix+"/:category/:id", a.makeHandler(handleRetrieve))
```

Comment:
A validation is missing in the option. Api prefix must always start by /

Reply:
Ok. Added validations and tests for the validations.